shared/fs/remote-container.js

```diff
@@ -2,17 +2,17 @@
 import * as Types from '../constants/types/fs'
 
 type RemoteTlfUpdates = {
-  tlf: Types.Path,
-  writer: string,
   timestamp: number,
+  tlf: Types.Path,
   updates: Array<Types.Path>,
+  writer: string,
 }
 
-const GetRowsFromTlfUpdates = (tlfUpdates: Types.UserTlfUpdates): Array<RemoteTlfUpdates> => tlfUpdates.toArray().map(t => ({
-  tlf: t.path,
-  writer: t.writer,
+const GetRowsFromTlfUpdate = (t: Types.TlfUpdate): RemoteTlfUpdates => ({
```

Comment:
I notice this is still sending when nothing has changed. Should we add a memoization layer?

Reply:
really the serializer needs to be updated. kbfs team was working on this while i was doing stuff so i didn't bother

Reply:
@buoyad it shouldn't be...that function is only called in `mergeProps`.

Reply:
@jzila `mergeProps`? It's in `serializer` [here](https://github.com/keybase/client/blob/b98e9e4760d33dc0fb8793242ae2ebc4bd5d6eab/shared/menubar/remote-serializer.desktop.js#L32). When I have `debugSerializer` on I see lists for key `fileRows` output to console